master/internal/core.go

```diff
@@ -73,6 +73,10 @@ func New(version string, logStore *logger.LogBuffer, config *Config) *Master {
 	}
 }
 
+func (m *Master) getConfig(c echo.Context) (interface{}, error) {
+	return m.config.Printable()
```

Comment:
non-blocking: Can we add this to the master info endpoint instead?

Reply:
That would also work, of course -- why do you think it would be preferable?

Reply:
It seems to fit that endpoint already and would be preferable to adding more endpoints.

Reply:
Yeah, I thought about that, but I felt like (a) none of the existing clients of that endpoint care about this information (b) the client for this new endpoint doesn't care about the stuff in `/info` (c) the cost of a new endpoint is pretty minimal. So I'm inclined to just add a new endpoint.